docs/DeveloperGuide.md

```diff
@@ -164,40 +164,40 @@ This section describes some noteworthy details on how certain features are imple
 
```

Comment:
Many references to ____AddressBook___ within this section, do you think it's worth changing?